From 0ba0986f1ebc5ac1ad0a3d51e8039dbdccb31c25 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Thu, 14 Nov 2024 07:33:56 +0000
Subject: [PATCH 1/2] Makefile: Let bootchartd.conf use EARLY_PREFIX

So that bootchartd.conf can install to correct location when cross build.

Upstream-Status: Submitted [https://github.com/xrmx/bootchart/pull/105]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b99b141..99a3ec7 100644
--- a/Makefile
+++ b/Makefile
@@ -100,7 +100,7 @@ install-chroot:
 
 install-collector: all install-chroot
 	install -m 755 -D bootchartd $(DESTDIR)$(EARLY_PREFIX)/sbin/$(PROGRAM_PREFIX)bootchartd$(PROGRAM_SUFFIX)
-	install -m 644 -D bootchartd.conf $(DESTDIR)/etc/$(PROGRAM_PREFIX)bootchartd$(PROGRAM_SUFFIX).conf
+	install -m 644 -D bootchartd.conf $(DESTDIR)$(EARLY_PREFIX)/etc/$(PROGRAM_PREFIX)bootchartd$(PROGRAM_SUFFIX).conf
 	install -m 755 -D bootchart-collector $(DESTDIR)$(PKGLIBDIR)/$(PROGRAM_PREFIX)bootchart$(PROGRAM_SUFFIX)-collector
 
 install-docs:
-- 
2.44.1

